dist/injector.js

```diff
@@ -10,39 +10,20 @@
  *
  *
  */
-window.NS = window.NS || {};
-window.NS.bundleHost = window.NS.bundleHost || '/';
+window.FAI = window.NS || {};
```

Comment:
Almost... you need to set `window.FAI = window.FAI || {};`. You will need to update the reference in /example/index.html.